src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import Todo from '../../types/Todo';
+import users from '../../api/users';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const currentUser = users.find(user => user.id === todo.userId);
+
+  return (
+    <article
+      data-id={todo.id}
+      className={`TodoInfo${todo.completed ? ' TodoInfo--completed' : ''}`}
+    >
+      <h2 className="TodoInfo__title">
+        {todo.title}
+      </h2>
+
+      <a className="UserInfo has-text-grey" href={`mailto:${currentUser?.email}`}>
+        {currentUser?.name}
+      </a>
```

Comment:
[checklist 4](https://github.com/mate-academy/react_add-todo-form/blob/master/newChecklist.md)
You can make a check for user and prevent rendering this part at all if there is no one
```suggestion
      {currentUser &&  (
        <a className="UserInfo has-text-grey" href={`mailto:${currentUser.email}`}>
          {currentUser.name}
        </a>
      )
```